### PR TITLE
Cluster Settings: show update errors using tooltips

### PR DESCRIPTION
**Pull request by spadgett**
/assign @rhamilto 
cc @wking 

Screen Shot 2019-09-23 at 9 32 03 AM

Screen Shot 2019-09-23 at 9 26 44 AM

Screen Shot 2019-09-23 at 9 15 44 AM

### CODE REVIEW

File: frontend/public/components/cluster-settings/cluster-settings.tsx [hunk 1]
```diff
@@ -129,10 +145,12 @@ export const CurrentVersion: React.SFC<CurrentVersionProps> = ({cv}) => {
   if (status === ClusterUpdateStatus.UpToDate || status === ClusterUpdateStatus.UpdatesAvailable) {
     return desiredVersion
       ? <span className="co-select-to-copy">{desiredVersion}</span>
-      : <React.Fragment><YellowExclamationTriangleIcon />&nbsp;Unknown</React.Fragment>;
+      : <><YellowExclamationTriangleIcon />&nbsp;Unknown</>;
```
Comment (rhamilto) [inline frontend/public/components/cluster-settings/cluster-settings.tsx]: Nit:  you're using `<React.Fragment>` above (as are the previously existing examples).

**Comment by spadgett** [comment]
/hold
for #2817

File: frontend/public/components/cluster-settings/cluster-settings.tsx [hunk 2]
```diff
@@ -88,22 +90,36 @@ const UpdatingMessage: React.SFC<CVStatusMessageProps> = ({cv}) => {
...
+    <Link to="/settings/cluster/clusteroperators">View details</Link>
+  </React.Fragment>;
+};
 
 const UpToDateMessage: React.SFC<{}> = () => <span>
   <GreenCheckCircleIcon /> Up to date
 </span>;
 
 export const UpdateStatus: React.SFC<UpdateStatusProps> = ({cv}) => {
+  if (cv.spec.channel === 'NoChannel') {
```
Comment (spadgett) [inline frontend/public/components/cluster-settings/cluster-settings.tsx]: @wking is `NoChannel` a thing? i can't find any references to it anywhere

Comment (wking) [reply frontend/public/components/cluster-settings/cluster-settings.tsx]: It's [a condition][1], not part of the `spec`.

[1]: https://github.com/openshift/cluster-version-operator/blob/59601a89bb7a65cf2fbaa4697d5142091f68534f/pkg/cvo/availableupdates.go#L146

Comment (spadgett) [reply frontend/public/components/cluster-settings/cluster-settings.tsx]: Thanks, I've implemented this incorrectly then. Let me fix :)

**Comment by spadgett** [comment]
/hold cancel

@rhamilto updated, another look?

**Comment by wking** [comment]
Can we get a bug for this PR where QE can verify with [this patch][1]?  Then we can clone/cherry-pick back to 4.2.

[1]: https://github.com/openshift/openshift-docs/pull/16678#discussion_r326876856

**Pull request merged**